src/custom/ecospheres/views/HomeView.vue

```diff
@@ -31,94 +36,320 @@ useHead({
 
 <template>
   <div class="datagouv-components">
-    <section class="fr-container fr-py-12v">
-      <div class="fr-grid-row fr-grid-row--gutters">
-        <div class="fr-col">
-          <h1>{{ homepageTitle }}</h1>
-          <p>
-            <i>ecologie</i><strong>.data.gouv</strong><i>.fr</i> référence et
-            centralise les données de la transition écologique.
-          </p>
-          <p>
-            Le catalogue rassemble les données utiles au déploiement des
-            politiques publiques portées par le Ministère de la Transition
-            écologique et de la Cohésion des territoires.
-          </p>
-        </div>
-        <div class="fr-col-md-6 fr-hidden fr-unhidden-md fr-grid-row--center">
-          <img
-            src="../assets/home-global-schema.svg"
-            alt=""
-            class="illustration"
-            width="441"
-            height="290"
-          />
-        </div>
+    <section class="fr-container fr-pt-12v">
+      <h1 class="main-title-v2">
+        Le&nbsp;catalogue des&nbsp;données
+        <br />
+        pour la <span class="highlight">transition écologique</span>
+      </h1>
+      <div class="big-search">
+        <SearchComponent
+          id="big-select-search"
+          placeholder="Rechercher"
+          search-label="Rechercher"
+        />
+        <p for="select-search" class="text-grey-300 fr-text--sm">
```

Comment:
L'attribut `for` n'a aucun effet ici.
Il faudrait pouvoir le raccrocher à l'input du `Multiselect` mais je n'ai pas encore trouvé comment…

Reply:
J'avoue être un peu perplexe concernant le label invisible, et je serai pour faire sauter cette attribut `for` sur ce paragraphe pour l'instant au moins

Reply:
Le label invisible est nécessaire pour coller à la maquette (champ de recherche sans label) et pour rester accessible (instruction d'utilisation du champ de recherche).